### PR TITLE
Pairplot of two variables with bokeh

### PR DESCRIPTION
**Pull request by agustinaarroyuelo**
## Description
This PR addresses issue #1169. 
bokeh_plot

## Checklist
- [x] Follows [official](https://github.com/arviz-devs/arviz/blob/master/CONTRIBUTING.md#pull-request-checklist) PR format
- [x] Includes a sample plot to visually illustrate the changes (only for plot-related functions)
- [x] Code style  correct (follows pylint and black guidelines)
- [x] Changes are listed in [changelog](https://github.com/arviz-devs/arviz/blob/master/CHANGELOG.md#v0xx-unreleased)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +19,7 @@
 * New grayscale style. This also add two new cmaps `cet_grey_r` and `cet_grey_r`. These are perceptually uniform gray scale cmaps from colorcet (linear_grey_10_95_c0) (#1164)
 * Add warmup groups to InferenceData objects, initial support for PyStan (#1126) and PyMC3 (#1171)
```
Comment (canyon289) [inline CHANGELOG.md:21]: Being picky here. Looks like theres a blank line that can be removed

File: arviz/plots/backends/bokeh/pairplot.py [hunk 2]
```diff
@@ -179,7 +185,7 @@ def get_width_and_height(jointplot, rotate):
                     row_ax.append(None)
                 else:
                     jointplot = row == col and numvars == 2 and marginals
-                    rotate = n == 1
+                    rotate = col == 1
```
Comment (canyon289) [inline arviz/plots/backends/bokeh/pairplot.py:188]: Thanks, much easier to read now

File: CHANGELOG.md (hunk 1 above)
Comment (canyon289) [reply CHANGELOG.md:21]: Is this changelog item correct? It feels like this PR is internally facing readability changes (which is great) but the feature described here ie something else?

Comment (agustinaarroyuelo) [reply CHANGELOG.md:21]: It was a duplicated line. Thank you for your helpful comment

File: CHANGELOG.md [hunk 3]
```diff
@@ -19,7 +19,9 @@
...
 
+* Integrate jointplot into pairplot, add point-estimate and overlay of plot kinds #1079
```
Comment (OriolAbril) [inline CHANGELOG.md:22]: this line has been triplicated, it is also 3 points above and the 3rd point starting from the top

**Comment by canyon289** [comment]
Ship it!

**Comment by canyon289** [comment]
Ah shoot I merged a little too quickly, which is fine. Next time do a rebase on master so the changes from master dont appear in this PR.

Ill slow down next time, my bad

**Comment by canyon289** [comment]
Basically thing change shouldnt have appeared in set of commits

image

**Pull request merged**